test/acceptance/behave/components/ngsiv2/entities/list_attribute_by_id.feature

```diff
@@ -0,0 +1,745 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+
+Feature: get an attribute by entity ID in NGSI v2. "GET" - /v2/entities/<entity_id>/attrs/<attr_name>
+  As a context broker user
+  I would like to get an attribute by entity ID in NGSI v2
+  So that I can manage and use them in my scripts
+
+  BackgroundFeature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file and restart service
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  @happy_path
+  Scenario:  get an attribute by entity ID in NGSI v2
+    Given  a definition of headers
+      | parameter          | value              |
+      | Fiware-Service     | test_id_happy_path |
+      | Fiware-ServicePath | /test              |
+      | Content-Type       | application/json   |
+    When create "1" entities with "3" attributes
+      | parameter        | value       |
+      | entities_type    | house       |
+      | entities_id      | room        |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+      | attributes_type  | celcius     |
+      | metadatas_number | 2           |
+      | metadatas_name   | very_hot    |
+      | metadatas_type   | alarm       |
+      | metadatas_value  | hot         |
+    And verify that receive several "Created" http code
+    When get an attribute "temperature_0" by ID "room"
+    Then verify that receive an "OK" http code
+    And verify that the attribute by ID is returned
+
+  @more_entities
```

Comment:
Thanks, Yes, I use generic tags as `skip`, `happy_path`, `only_develop`, etc, but each scenario has its specific tags and in case of Issue associated too, they are cheap... :-)